CONTRIBUTING.md

```diff
@@ -97,16 +97,14 @@ docker compose exec -it client /bin/sh
 To start the local Firezone cluster, follow these steps:
 
 ```
-docker compose build
-docker compose up -d postgres
```

Comment:
`elixir` already depends on the postgres container, I think this is redundant.

Reply:
👍